### PR TITLE
Feat/clear app data from developer menu

### PR DESCRIPTION
**Pull request by ghassenbenzahra123**
📝 **Summary**
---
**This PR fixes**  #1083  

**Description**
---
it adds an option to clear data & cache from dev menu.

**Tests**
---
🧪 **Use case 1**
---
💬 **Description:**

Launch the app and run option 6 on the developer menu.
 
📷 **Screenshots or GIFs (if applicable):**

[cleardata.webm](https://github.com/mawaqit/android-tv-app/assets/35707318/94cae5f7-1c01-4c10-b923-10e75f96fd6f)

**Checklist:**
---
- [x] **Coding Standards:** I have reviewed my code to ensure it follows the project's coding standards.
- [x] **Testing:** I have tested the changes and they work as expected.
- [x] **Merge Conflicts:** I have resolved any merge conflicts with the latest main/development branch.
- [x] **Branch Status:** The branch is up-to-date with the target branch (main/development).

### CODE REVIEW

**Review by YassinNouh21 — APPROVED** [review]
Great Work Ghassan

**Comment by YassinNouh21** [comment]
@ghassenbenzahra123  just check the conflict

File: android/app/src/main/kotlin/com/flyweb/MainActivity.kt [hunk 1]
```diff
@@ -35,4 +42,26 @@ class MainActivity : FlutterActivity() {
...
+private fun clearDataRestart(): Boolean {
+    try {
+        val processBuilder = ProcessBuilder()
+        processBuilder.command("sh", "-c", """
+            pm clear com.mawaqit.androidtv
+        """.trimIndent())
+        val process = processBuilder.start()
+        val exitCode = process.waitFor()
+        if (exitCode == 0) {
+            triggerRestart(this)
```
Comment (YassinNouh21) [inline android/app/src/main/kotlin/com/flyweb/MainActivity.kt:54]: the code does not compile because of `Unresolved reference: triggerRestart`

**Review by YassinNouh21 — CHANGES_REQUESTED** [review]
please check the compilation error

**Review by YassinNouh21 — APPROVED** [review]
@ghassenbenzahra123 the logic error still persist. I solved it by my side. Check if it works correctly or not

**Comment by ghassenbenzahra123** [comment]
> @ghassenbenzahra123 the logic error still persist. I solved it by my side. Check if it works correctly or not

It works thanks for the fix Yassin

**Pull request merged**